hashintel/hash
# Re-implement `createEntity` BP hook

## 🌟 What is the purpose of this PR?




This PR re-implements the BP `createEntity` hook using the new graph service.

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1202953520344821/1203109184120791/f) _(internal)_

## 🔍 What does this change?




See commits.

## 📜 Does this require a change to the docs?



The BP method documentation will need to be updated for the new type system once their definitions are stable.

## 🛡 What tests cover this?



None.

## ❓ How to test this?



1. Checkout the branch
2. Run the external services and `yarn dev`
3. Go to `localhost:3000` and log in
4. Go to `localhost:3000/entity-editor`
5. Click the "Create Entity" button (which uses the BP hook), and check the console to inspect the created entity

## 📹 Demo



https://user-images.githubusercontent.com/42802102/194828638-6b08d217-4307-4d23-8486-c16601ca2e59.mov
